presto-orc/src/main/java/com/facebook/presto/orc/DwrfStreamReorderingConfig.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.orc;
+
+import com.facebook.presto.orc.proto.DwrfProto;
+
+import java.util.List;
+import java.util.Map;
+
+import static com.google.common.base.MoreObjects.toStringHelper;
+import static java.util.Objects.requireNonNull;
+
+public class DwrfStreamReorderingConfig
```

Comment:
Would it be ok to use Ordering instead of Reordering?

or

We use StreamLayout class to rearrange streams, I guess we will create something like FlatMapStreamLayout for flat map streams. In this case FlatMapStreamLayout[Options|Config] sounds about right. What do you think?

Reply:
Yes, my plan is to add another method to StreamLayout interface which accepts the DwrfStreamReorderingConfig config in addition to the list of data streams. This method is used when the optional DwrfStreamReorderingConfig is non empty

Will become clearer when I send the next PR